tests/caasp/register_and_check.pm

```diff
@@ -22,7 +22,7 @@ sub run {
     }
 
     # Check that registration was succeeded
-    assert_script_run 'SUSEConnect --status-text | grep -o "Status: ACTIVE"';
+    assert_script_run 'SUSEConnect --status-text | tee /dev/tty | grep -q "Status: ACTIVE"';
```

Comment:
maybe https://superuser.com/a/914862/327890 is what you want:

```perl
assert_script_run 'SUSEConnect --status-text | tee /dev/tty | grep -z -q "Status: ACTIVE"';
```

Reply:
I welcome the idea, but I don't see difference in functionality when I try your example manually

Reply:
Please explain, I merged so panos can analyze failures meanwhile..